packages/components/ItemFields/ItemHeader.js

```diff
@@ -43,28 +58,44 @@ export const ItemHeader = ({ item }) => {
 
   return (
     <ColumnHeader>
-      <PathButton
-        color="white"
-        onClick={() => {
-          console.log('Change path');
-        }}
-      >
-        Personal
-        <Delimeter>|</Delimeter>
-        Passwords
-      </PathButton>
-      <ActionButton
-        icon="share"
-        color="white"
-        onClick={() => {
-          console.log('Share');
-        }}
-      />
-      <ActionButton
-        icon={favorite ? 'favorite-active' : 'favorite'}
-        color="white"
-        onClick={handleToggleFavorites}
-      />
+      {isTrashItem ? (
+        <>
+          <Empty />
+          <ActionButton color="white" onClick={onClickRestoreItem}>
+            Restore
+          </ActionButton>
+          <ActionButton
+            icon="trash"
+            color="white"
+            onClick={onClickRemoveItem}
+          />
+        </>
+      ) : (
+        <>
+          <PathButton
+            color="white"
+            onClick={() => {
+              console.log('Change path');
```

Comment:
hm?

Reply:
I implemented this functional in next pr.
So this console is absent now

Reply:
please use the notification (not-yet-implemented) instead of the console.log